x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -137,7 +137,6 @@ const securitySubPlugins = [
   `${APP_ID}:${SecurityPageName.network}`,
   `${APP_ID}:${SecurityPageName.timelines}`,
   `${APP_ID}:${SecurityPageName.case}`,
-  `${APP_ID}:${SecurityPageName.administration}`,
```

Comment:
Just curious, why do you need to remove this?

Reply:
Hey @dasansol92, this is a list of registered plugins, with this PR "administration" is no longer a plugin, it is not registered to the core plugins as `"${APP_ID}:${SecurityPageName.administration}"` anymore. 
But it is included as a [deepLink](https://github.com/elastic/kibana/pull/102685/files#diff-834012aeafcf19afae6d63532fca7529d58dfed45c7cc4bd1d356c0d9f63b8d7) of the main SecuritySolutions (`APP_ID`) plugin.
And we have our [own main router](https://github.com/elastic/kibana/pull/102685/files#:~:text=x-pack/plugins/security_solution/public/app/index.tsx) now.